src/Agent/NewRelic/Agent/Core/Core.csproj

```diff
@@ -76,8 +76,8 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
     <PackageReference Include="Grpc.Net.Client" Version="2.71.0" />
-    <PackageReference Include="Microsoft.Extensions.Configuration" Version="[2.0.0]" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="[2.0.0]" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="[9.0.8]" />
```

Comment:
This seems like a huge jump in version.  Presumably we are currently referencing such an old version for a reason.

Reply:
Of course, if Dependabot would just correctly respect NuGet package reference conventions, this would be a non-issue. The square brackets mean "require this specific version" and are never subject to automatic update.

Reply:
PR to fix the bug: https://github.com/newrelic/newrelic-dotnet-agent/pull/3230